pages/docs/react/latest/introduction.mdx

```diff
@@ -16,7 +16,7 @@ All our documented examples can be compiled in our [ReScript Playground](/try) a
 
 - No Babel plugins required (JSX is part of the language!)
 - Comes with all essential React APIs for building production ready apps (`useState`, `useReducer`, `useEffect`, `useRef`,...)
-- No component class API (all ReScript & React codebases are built on functional components & hooks)
+- No component class API (all ReScript & React codebases are built on function components & hooks)
 - Strong level of type safetiness and type inference for component props and state values
 - [GenType](/docs/gentype/latest/introduction) support for importing / exporting React components in Flow and TypeScript codebases
```

Comment:
Should we still mention Flow here?

Reply:
No it's not supported in v11.

Reply:
removed the flow support https://github.com/rescript-association/rescript-lang.org/pull/593/commits/afd6833299b3fcb1d38025b903b2f0100a84f853